### DISCUSSION TITLE
I suggest a few features

### DISCUSSION THREAD

**Discussion by Runa-bit** [stoatchat/discussions#784] [Feedback]
1. Add avatar circling when talking
2. To display the number of people in the voice channels and see who is sitting there
3. Fix the resolution in the voice settings

**Comment by Runa-bit** [comment]
4.Add screen demo with sound!!!(this is important)

**Reply by Archmaner** [reply]
+

**Comment by Runa-bit** [comment]
5.add a microphone and sound button at the bottom where you are (to mute).

**Comment by amycatgirl** [comment]
All of these features are either already implemented or in the process of being added to the client/voice server rewrite.

There are no ETAs though

**Comment by Runa-bit** [comment]
6.add the possibility of uploading gifs of the tenor to the chat room.

**Reply by Archmaner** [reply]
bro you read thoughts

**Reply by amycatgirl** [reply]
This will be added in a future update, but instead of using tenor, Revolt will use GifBox, a gif hosting platform built by Revolt themselves.

**Reply by ab918** [reply]
is there a reason that they are not using tenor?
